Tests/TestData/spy_pivot_pnt_hl.txt

```diff
@@ -0,0 +1,293 @@
+Date,Open,High,Low,Close,Volume,PPHL
+2020-01-02T14:30:00Z,323.54,324.89,322.53,324.87,59253833,0
```

Comment:
Where does this test data come from? Does it satisfy the LEAN requirement of a third-party export/validation?

Reply:
It might be nice to chart these in excel to highlight the pivots working.